### DISCUSSION TITLE
PersistentWindows-Debloated

### DISCUSSION THREAD

**Discussion by kangyu-california** [kangyu-california/PersistentWindows#328] [General]
For users who do not want webpage-commander feature, you may try https://www.github.com/RaymondTracer/PersistentWindows-Debloated, which seems to be a trustworthy repo.

The author does not provide a download link though, here is my build from that code.
[PersistentWindows5.54_NoWebCommander.zip](https://github.com/user-attachments/files/15778655/PersistentWindows5.54_NoWebCommander.zip)

Note that the "debloated" build still check upgrade from my repo, you may want to run PW with  -check_upgrade=0

The reason I insist on integrating the feature into PW can be found in #314

Thanks for all the trust, support and bug report you have contributed to PW.

**Comment by libove** [comment]
I hadn't even been aware of this bloat until the 5.73 changelog note about the option to disable it.
Please, don't bloat. Persistent Windows does one thing well. Please don't add any bloat to it. Thanks.
